Lib/test/test_asyncio/test_streams.py

```diff
@@ -9,15 +10,10 @@
 import threading
 import unittest
 from unittest import mock
-from test.support import socket_helper
-try:
-    import ssl
-except ImportError:
-    ssl = None
-
-import asyncio
+from test.support import import_helper, socket_helper
 from test.test_asyncio import utils as test_utils
 
+ssl = import_helper.import_module('ssl')
```

Comment:
This is a change in behavior. It raises SkipTest if the ssl module cannot be imported.

Reply:
Thank you, I didn't know that. Reverted.